cmds/builtin/builtin.go

```diff
@@ -116,13 +111,12 @@ func main() {
 	// There is no FIXME
 	if false {
```

Comment:
Could we get rid of this?

You could make an issue instead containing the code.